dts/ti/mspm0/g/mspm0g51xx-pinctrl.dtsi

```diff
@@ -0,0 +1,2389 @@
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_ANALOG)>;
+			};
+
+			/omit-if-no-ref/ gpioa_26_pa26: gpioa_26_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_GPIO)>;
+			};
+
+			/omit-if-no-ref/ uc0_sda_pa26: uc0_sda_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_3)>;
+			};
+
+			/omit-if-no-ref/ uc0_tx_pa26: uc0_tx_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_3)>;
+			};
+
+			/omit-if-no-ref/ uc1_scl_pa26: uc1_scl_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_4)>;
+			};
+
+			/omit-if-no-ref/ uc1_rx_pa26: uc1_rx_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_4)>;
+					input-enable;
+			};
+
+			/omit-if-no-ref/ timg7_ccp0_pa26: timg7_ccp0_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_5)>;
+			};
+
+			/omit-if-no-ref/ sysctl_fcc_in_pa26: sysctl_fcc_in_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_6)>;
+					input-enable;
+			};
+
+			/omit-if-no-ref/ tima_fault0_pa26: tima_fault0_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_7)>;
+			};
+
+			/omit-if-no-ref/ uc3_pico_pa26: uc3_pico_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_8)>;
+			};
+
+			/omit-if-no-ref/ uc3_tx_pa26: uc3_tx_pa26 {
+					pinmux = <MSP_PINMUX(58, MSPM0_PIN_FUNCTION_8)>;
+			};
+
+			/omit-if-no-ref/ usbfs0_dp_pa27: usbfs0_dp_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_ANALOG)>;
+			};
+
+			/omit-if-no-ref/ gpioa_27_pa27: gpioa_27_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_GPIO)>;
+			};
+
+			/omit-if-no-ref/ uc0_scl_pa27: uc0_scl_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_3)>;
+			};
+
+			/omit-if-no-ref/ uc0_rx_pa27: uc0_rx_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_3)>;
+					input-enable;
+			};
+
+			/omit-if-no-ref/ uc1_sda_pa27: uc1_sda_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_4)>;
+			};
+
+			/omit-if-no-ref/ uc1_tx_pa27: uc1_tx_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_4)>;
+			};
+
+			/omit-if-no-ref/ timg7_ccp1_pa27: timg7_ccp1_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_5)>;
+			};
+
+			/omit-if-no-ref/ sysctl_clk_out_pa27: sysctl_clk_out_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_6)>;
+			};
+
+			/omit-if-no-ref/ lfss_rtc_out_pa27: lfss_rtc_out_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_7)>;
+			};
+
+			/omit-if-no-ref/ comp0_out_pa27: comp0_out_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_8)>;
+			};
+
+			/omit-if-no-ref/ uc3_sclk_pa27: uc3_sclk_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_9)>;
+			};
+
+			/omit-if-no-ref/ uc3_rx_pa27: uc3_rx_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_9)>;
+					input-enable;
+			};
+
+			/omit-if-no-ref/ tima_fault2_pa27: tima_fault2_pa27 {
+					pinmux = <MSP_PINMUX(59, MSPM0_PIN_FUNCTION_10)>;
+			};
+
+			/omit-if-no-ref/ gpioa_00_pa0: gpioa_00_pa0 {
+					pinmux = <MSP_PINMUX(1, MSPM0_PIN_FUNCTION_GPIO)>;
```

Comment:
Please make the pinconfig re-order with pin number ascending.
As mentioned in https://github.com/zephyrproject-rtos/hal_ti/pull/76, this file must be delta with baseline.

Reply:
^^ @d-philpot